patches/express-graphql+0.12.0.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/express-graphql/index.js b/node_modules/express-graphql/index.js
+index 3bc78dc..477be08 100644
+--- a/node_modules/express-graphql/index.js
++++ b/node_modules/express-graphql/index.js
+@@ -153,6 +153,10 @@ function graphqlHTTP(options) {
+                     result = { ...result, extensions };
+                 }
+             }
++            // If errors in execution, set the `Cache-Control: no-cache` header.
++            if (result.errors != null) {
```

Comment:
I thought about being fancier and augmenting the wrapper we use to pass these in as a callback, `onErrorPostProcess`, or `customHeaderFn` so if we want to iterate on this via:
  - change the kinds of errors that lead to us not caching
  - change headers we set in response
  - generally any increased complexity in one of these two ^

it might be nicer to work that way than needing to modify the patch.

But, I figured that's over-engineering for now, if we ever have to come back to this patch for iteration or tweaking, can maybe improve it that way.